index.html

```diff
@@ -34,12 +34,236 @@ <h3>I am a software developer</h3>
+                            <p class="see1">See Project</p>
+                        </section>
+                    </section>
+                </section>
+                <section class="card-works">
+                    <section class="mask">
+                        <section class="img-placeholder">
+                            <section class="div">
+                                <section class="right-works">
+                                    <p class="pro">Profesional Art Printing Data</p>
+                                    <p class="sub1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                                    <ul class="tags">
+                                        <li class="tags1"><h6>html</h6></li>                                            
+                                        <li class="tags2"><h6>bootstrap</h6></li>
+                                        <li class="tags3"><h6>Ruby</h6></li>
+                                    </ul>
+                                </section>
+                            </section>
+                        </section>
+                        <!-- <section class="mask-sub"></section> -->
+                    </section>
+                    <section class="action">
+                        <section class="but">
+                            <p class="see1">See Project</p>
+                        </section>
+                    </section>
+                </section>
+                <section class="card-works">
+                    <section class="mask">
+                        <section class="img-placeholder">
+                            <section class="div">
+                                <section class="right-works">
+                                    <p class="pro">Profesional Art Printing Data</p>
+                                    <p class="sub1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                                    <ul class="tags">
+                                        <li class="tags1"><h6>html</h6></li>                                            
+                                        <li class="tags2"><h6>bootstrap</h6></li>
+                                        <li class="tags3"><h6>Ruby</h6></li>
+                                    </ul>
+                                </section>
+                            </section>
+                        </section>
+                        <!-- <section class="mask-sub"></section> -->
+                    </section>
+                    <section class="action">
+                        <section class="but">
+                            <p class="see1">See Project</p>
+                        </section>
+                    </section>
+                </section>
+            </section>
+            <section class="about">
```

Comment:
- [ ] As the last code reviewer said: Kindly add spacings in the about section to match the design template. 👇🏽 

- **Your template**
![image](https://user-images.githubusercontent.com/58642949/192076216-f52b33f9-96fe-452f-badb-f13b5dcd457e.png)

-**Figma template**
![image](https://user-images.githubusercontent.com/58642949/192076255-c9c6ca38-4e03-463c-bf02-d1edf786b551.png)